index.html

```diff
@@ -230,9 +230,9 @@ <h3>Skills</h3>
       </p>
 
       <form class="form-page">
```

Comment:
- [ ] The form should use the `action` and `method` attribute to integrate with [formspree service](https://formspree.io/html).